shopinvader_api_quotation/schemas/sale.py

```diff
@@ -0,0 +1,28 @@
+from extendable_pydantic import StrictExtendableBaseModel
+
+from odoo.addons.shopinvader_schema_sale.schemas.sale import Sale
+
+
+class Sale(Sale, extends=True):
+    available_for_quotation: bool | None = None
+    shop_only_quotation: bool | None = None
+    customer_ref: str | None = None
```

Comment:
It's a little strange to introduce the customer ref into this addon since its usage seems to me wider than the quotation context.

Reply:
We can move this it the sale schema and also add the possibility to set it from the cart API